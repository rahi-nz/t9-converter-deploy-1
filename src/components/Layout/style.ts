import styled from 'styled-components';

export const Container = styled.section`
  text-align: center;
`;

export const Content = styled.section`
  width: 240px;
`;
