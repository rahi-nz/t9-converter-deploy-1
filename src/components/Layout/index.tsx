import Head from 'next/head';
import { Container, Content } from './style';

interface Props {
  children: React.ReactNode;
}

const Layout = ({ children }: Props) => {
  return (
    <>
      <Head>
        <title>T9 converter</title>
        <link rel="shortcut icon" href="/favicon/favicon.ico" />
        <link rel="shortcut icon" href="/favicon/favicon-16x16.png" />
        <link rel="shortcut icon" href="/favicon/favicon-32x32.png" />
        <link rel="shortcut icon" href="/favicon/favicon-96x96.png" />
        <link
          href="https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap"
          rel="stylesheet"
        />
      </Head>
      <Container>
        <Content>{children}</Content>
      </Container>
    </>
  );
};
export default Layout;
