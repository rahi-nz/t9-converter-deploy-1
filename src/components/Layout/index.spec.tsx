import renderer from 'react-test-renderer';
import Layout from './index';

describe('<Layout/>', () => {
  let rootElement;
  let testRenderer;

  // eslint-disable-next-line prefer-const,react/display-name
  rootElement = () => (
    <Layout>
      <div></div>
    </Layout>
  );
  // eslint-disable-next-line prefer-const
  testRenderer = renderer.create(rootElement());
  it('Should render correctly', () => {
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
