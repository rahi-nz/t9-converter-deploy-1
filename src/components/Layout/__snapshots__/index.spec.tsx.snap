// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout/> Should render correctly 1`] = `
<section
  className="style__Container-ph9pdk-0 hbgMfM"
>
  <section
    className="style__Content-ph9pdk-1 goArNV"
  >
    <div />
  </section>
</section>
`;
